src/tests/Common/scripts/run-pmi-diffs.py

```diff
@@ -1,719 +0,0 @@
-#!/usr/bin/env python
```

Comment:
cc: @dotnet/jit-contrib 

Is this file just no longer in use?

Reply:
These are the Jenkins CI leftovers. None of these scripts are compatible with dotnet/runtime. https://github.com/dotnet/runtime/pull/74598#discussion_r960126130